charts/otomi-db/values.yaml

```
@@ -3,7 +3,7 @@ labels:
   prometheus: system
   helm.sh/resource-policy: keep
 instances: 2
-imageName: ghcr.io/cloudnative-pg/postgresql:15.3
+imageName: ghcr.io/cloudnative-pg/postgresql:17.0
```

17.2 is currently the latest. I think its best to always pick the latest patch version.
```suggestion
imageName: ghcr.io/cloudnative-pg/postgresql:17.2
```

---

see: https://github.com/cloudnative-pg/postgres-containers/pkgs/container/postgresql